pkgs/development/python-modules/pythonparser/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "pythonparser";
+  version = "1.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qby6w2pcfwxi4w62rly08d9wj32amf6m7rmbrqw0xz6mirq7rrp";
+  };
+
+  patches = [
+    ./no_test_parser_teardown.patch
+  ];
+
+  propagatedBuildInputs = [
+    regex
+  ];
+
+  meta = {
+    description = "A Python parser intended for use in tooling";
+    homepage = https://m-labs.hk/pythonparser;
```

Comment:
Please add
```
    downloadPage = https://github.com/m-labs/pythonparser;
```
because that one is really hard to find.

Reply:
✓